cli/azd/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 
 - Fixed an issue where passing `--help` to `azd` would result in an error message being printed to standard error before the help was printed.
 - [[#71]](https://github.com/Azure/azure-dev/issues/71) Fixed detection for disabled GitHub actions on new created repos.
+- [[#62]](https://github.com/Azure/azure-dev/issues/62) Fixed using template name and repository path.
```

Comment:
This isn't really a user facing thing and so I would vote that we don't call it out in the changelog and instead just focus on documenting the key bugs we fixed.  `git log` can always be used to see the complete list of changes.

Just my two cents.